packages/embarkjs/embarkjs/src/lib/index.js

```diff
@@ -6,7 +6,10 @@ import Utils from './utils';
 
 var EmbarkJS = {
   onReady: function (cb) {
-    Blockchain.execWhenReady(cb);
+    if (Blockchain.blockchainConnector) {
```

Comment:
As discussed in https://github.com/embark-framework/embark/pull/2096#issuecomment-559469008 instead of using `isAvailable()` we're checking if `blockchainConnector` is actually set.